.gitignore

```
@@ -1,4 +1,5 @@
 .vscode/
 .env
-/database/data
-.DS_Store
\ No newline at end of file
+/database/devdata
+/database/proddata
```

gitignoreされるようにハイフンを入れよう！
```suggestion
/database/dev-data
/database/prod-data
```